account-keys/slip10/README.md

```diff
@@ -1,5 +1,46 @@
-# MobileCoin SLIP-0010-Based Key Derivation
+# MobileCoin Key Derivation
 
-This crate provides utilities to handle SLIP-0010 key bytes and their relation to the MobileCoin [`AccountKey`](mc_account_keys::AccountKey) structure, which contains a pair of Ristretto255 view/spend private scalars.
+This crate implements a standard method for deriving a MobileCoin [`AccountKey`](mc_account_keys::AccountKey) from entropy using the following procedure:
 
-This also features a trait to create a Slip10Key from entropy and path, along with the canonical method of converting a BIP-39 [`Mnemonic`](tiny_bip32::Mnemonic) with a given BIP-32 path into a [`Slip10Key`](Slip10Key) usable within MobileCoin.
+1. A cryptographically secure random number generator is used to gather 32 bytes of entropy.
+1. These bytes are mapped into a BIP-39-compliant [`Mnemonic`](tiny_bip39::Mnemonic) seed phrase.
+1. The ([NFKD normalized](https://en.wikipedia.org/wiki/Unicode_equivalence#Normal_forms)) UTF-8 seed phrase is hashed using an empty-string password according to [the BIP-39 specification](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki#from-mnemonic-to-seed) into a [`Seed`](tiny_bip39::Seed).
+1. Using the [BIP-44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) compliant [BIP-32 path](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) `m/44'/866'/<account>'`, the seed bytes are hashed again, according to the [SLIP-0010 procedure for Ed25519 curves](https://github.com/satoshilabs/slips/blob/master/slip-0010.md) to deterministically generate the resulting child key.
+1. The resuting [`Slip10Key`](crate::Slip10Key) is expanded into the necessary Ristretto255 scalars used as CryptoNote view and spend keys using HKDF-SHA512, salted with the byte strings `b"mobilecoin-ristretto255-view"` and `b"mobilecoin-ristretto255-spend"`, respectively.
+
+## Usage
+
+The usage is fairly straightforward, you start with a mnemonic, convert to a [`Slip10Key`](crate::SLip10Key), and then on to an [`AccountKey`](mc_account_key::AccountKey).
+
+```
+use mc_account_keys::AccountKey;
+use mc_account_keys_slip10::Slip10KeyGenerator;
+use bip39::{Language, Mnemonic};
+
+// Create a new mnemonic from random bytes
+let mnemo = Mnemonic::new(Language::English);
+
+// FIXME: Let the user backup their phrase
+
+// Create a SLIP-0010 private child key for account "1"
+let slip10 = mnemo.derive_slip10_key(1)
+
+// Derive the AccountKey from the Slip10Key
+let account_key = AccountKey::from(slip10);
+
+// go forth and spend money!
+```
+
```

Comment:
```suggestion
Note: because all accounts are Ristretto255 points, accounts do not have to be "registered" with the Mobilecoin network. Once keys are generated, users in possession of those keys can immediately receive or spend Mobilecoin.

```